.vscode/tasks.json

```diff
@@ -2,10 +2,11 @@
 	"version": "2.0.0",
 	"tasks": [
 		{
-			"label": "dapr-VehicleApp-run",
-			"appId": "vehicleapp",
+			"label": "dapr-SeatAdjuster-run",
+			"appId": "seatadjuster",
```

Comment:
does it make sense to have a task for the SeatAdjuster but not for the other examples? Couldn't we have a general one?